executor.c

```diff
@@ -0,0 +1,90 @@
+//
+//  executor.c
+//  connectivi_graph
+//
+//  Created by Artem on 26/08/2022.
+//  Copyright © 2022 Artem. All rights reserved.
+//
+
+#include "finding_connectivity_components.h"
+
+
+Pair_int_int* read_grath_from_txt(char* file_path_name, int* points_cnt, int* arc_cnt){
+    Pair_int_int *v;
+    
+    FILE *in_file;
+    in_file = fopen(file_path_name, "r");
```

Comment:
There is not necessary to initialize variables in different statement.

Reply:
Ok